includes/definitions/saf-cfml4.yaml

```diff
@@ -0,0 +1,12 @@
+os: saf-cfml4
```

Comment:
Any reason this is a new OS rather than adding to the `saf` OS we have?

Reply:
Not having the same MIB across product(s) is fine. Do they not have other models within this range?

The short is, we don't want to be too specific with the OS names as it leads to duplicates and more work to keep thins up to date.

Reply:
Hello heroes, Yes I understanding the point. The OS I made supports a product line of 3 different models with 3 types of optional modules and 4 types of radios. That covers the whole product line of the L4. There will not come any other models as this product line has been surpassed by newer product lines. Best, Janno

Reply:
I think laf's point is it should probably just be saf-cfm right?

SAF's products are CFIP, CFQ, CFM

Reply:
Yes, I looked into this and you are right. So I will update all with a renamed os.

Reply:
I know changed the OS name from saf-cfml4 to saf-cfm.